edx_solutions_api_integration/courses/views.py

```diff
@@ -1857,6 +1869,11 @@ def get(self, request, course_id):  # pylint: disable=W0613
 
             if cohort_user_ids:
                 users_enrolled_qs = users_enrolled_qs.filter(id__in=cohort_user_ids)
+            if exclude_type and organization:
```

Comment:
We can put it under
`if organization:`
above

Reply:
Addressed